src/Native/System.Native/CMakeLists.txt

```diff
@@ -31,8 +31,17 @@ add_library(System.Native
     ${NATIVE_SOURCES}
 )
 
+
+add_library(System.Native.a
+    STATIC
+    ${NATIVE_SOURCES}
+)
+SET_TARGET_PROPERTIES(System.Native.a PROPERTIES PREFIX "")
```

Comment:
who disabled the prefix on the shared libraries? I am asking because it looks not having the prefix can cause some issues. I am seeing the following quote on the SO thread

> You can name one any way you want, but ld's -l assuming a lib prefix applies to both static and shared libraries and goes back a long way

so it looks to me it is better to keep the prefix for both static and dynamic libraries.